src/cascadia/UnitTests_TerminalCore/ConptyRoundtripTests.cpp

```diff
@@ -213,6 +215,9 @@ class TerminalCoreUnitTests::ConptyRoundtripTests final
 
     TEST_METHOD(NewLinesAtBottomWithBackground);
 
+    TEST_METHOD(WrapNewLineAtBottom);
+    TEST_METHOD(WrapNewLineAtBottomLikeMSYS);

```

Comment:
I just want to reiterate how happy I am that we're building tests like this that simulate our client applications and verify the desired behavior through the PTY.